live/channel.go

```diff
@@ -50,7 +61,21 @@ func (c Channel) String() string {
 	return ch
 }
 
-// IsValid checks if all parts of the address are valid.
+var channelPattern = regexp.MustCompile("^[A-Za-z0-9_-]*$")
+
+// IsValid checks if all parts of the Channel are valid.
 func (c *Channel) IsValid() bool {
-	return c.Scope != "" && c.Namespace != "" && c.Path != ""
+	allPartsExist := c.Scope != "" && c.Namespace != "" && c.Path != ""
+	if !allPartsExist {
+		return false
+	}
+	ok := channelPattern.MatchString(c.Scope)
+	if !ok {
+		return false
+	}
+	ok = channelPattern.MatchString(c.Namespace)
+	if !ok {
+		return false
+	}
+	return channelPattern.MatchString(c.Path)
```

Comment:
@FZambia -- the path pattern should also allow "/" and ".", and maybe "=" -- then it could handle most standard paths and base64 encoded values

Reply:
Added those ^^